sopel/config/types.py

```diff
@@ -741,7 +741,8 @@ def parse(self, value):
                     "Value must be an existing or creatable directory.")
         if not self.directory and not os.path.isfile(value):
             try:
-                open(value, 'w').close()
+                with open(value, 'w') as fd:
+                    fd.close()
```

Comment:
Extremely pedantic quibble: the name `fd` suggests that this is a file descriptor (i.e. an integer) rather than a file-like object. I usually use the identifier `f` for this operation. Not worth making a fuss over, except to the extent that this comment is making a fuss.

Reply:
This is fair. I've already thrashed GHA so much today, what's once more? Fixed.